test/OpenTelemetry.Exporter.Geneva.Benchmark/Exporter/MetricExporterBenchmarks.cs

```diff
@@ -547,7 +548,7 @@ public void InstrumentWithWithGenevaCounterMetricExporter4Dimensions()
     [Benchmark]
     public void SerializeCounterMetricItemWith3Dimensions()
     {
-        this.exporter.SerializeMetricWithTLV(
+        this.tlvMetricsExporter.SerializeMetricWithTLV(
```

Comment:
It would be better to create a dedicated TlvMetricExporter in the Benchmark setup instead of using `GenevaMetricExporter.Exporter`. This would make it easier to add and compare the protobuf serialization benchmarks in future.

You should be able to mark `GenevaMetricExporter.Exporter` private.

Reply:
@utpilla - This class has lot of helper methods that would be useful for protobuf benchmarks as well. I think we should move that to a common place. I can rename this one to TlvMetricExporterBenchmarks and create a different one for protobuf later. Let me know if that sounds ok to you, I can do follow up for that to keep this one small.

Reply:
> It would be better to create a dedicated TlvMetricExporter in the Benchmark setup instead of using GenevaMetricExporter.Exporter

done.